### DISCUSSION TITLE
How to configure second Authenticator app

### DISCUSSION THREAD

**Discussion by jur** [community/community#69862] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When I configure Microsoft Authentication App twice the first instance is not working any more. Why is it suggested to use a second method when it does not work?
Why is Google Authenticator not supported?
Why I am forced to use 2FA when nobody is using my code? When you are serious about that you must provide an Authentication App for the Sony Playstation 2.
Why does the support chat bot asks questions, but does not answer a question? (for example the questions above)

**Comment by Imranchowdhury500** [comment]
https://github.com/orgs/community/discussions/69862#discussion-5727367

**Reply by Imranchowdhury500** [reply]
https://github.com/orgs/community/discussions/69862#discussioncomment-7267521
